downstream/modules/platform/proc-account-linking.adoc

```diff
@@ -5,19 +5,43 @@
 [id="account-linking_{context}"]
 = Linking your account
 
- [TEXT]
+{PlatformNameShort} 2.5 provides a centralized location for users, teams and organizations to access the platform's services and features. When you upgrade from a previous version of {PlatformNameShort}, your existing account is automatically migrated to a single platform account. However, if you have multiple component accounts, your accounts must be linked to use the centralized features of the platform.
+
+The first time you log in to {PlatformNameShort} 2.5, the platform searches through the existing services to locate a user account with the credentials you entered. When there is a match to an existing account, that account is registered and becomes centrally managed by the platform. Any subsequent component accounts in the system are orphaned. This can cause problems accessing the services in {PlatformNameShort}. 
+
+To address this problem, you can use the account linking procedure to authenticate from any of your existing component accounts and still be recognized by the platform. Linking accounts will associate existing component accounts with the same user profile. 
```

Comment:
Suggestion: 
```suggestion
To address this problem, use the account linking procedure to authenticate from any of your existing component accounts and it will be recognized by the platform. Linking accounts associates existing component accounts with the same user profile. 
```

Reply:
Good edits. I made all the changes except for "it will" because I think it's supposed to be saying that going through the procedure from a particular account will let the platform recognize the account you're authenticating **from**. So, you can authenticate from an existing account and still be recognized. Does that sound right?

Reply:
Sure, go for it! I think it could go either way, but was trying to adhere to IBM Style guide when it says, "Do not use future tense in a cause and effect sentence, but you can use it for valid future situations." Sorry if that was confusing! Let me know when you are ready to squash and merge..